pages/services/cassandra/2.6.0-3.11.4/configuration/index.md

```diff
@@ -0,0 +1,281 @@
+---
+layout: layout.pug
+navigationTitle: Configuring 
+excerpt: Installation options, configuration regions, node settings, etc.
+title: Configuring Cassandra
+menuWeight: 42
+model: /services/cassandra/data.yml
+render: mustache
+---
+
+
+#include /services/include/configuration-install-with-options.tmpl
+#include /services/include/configuration-create-json-file.tmpl
+#include /services/include/configuration-service-settings.tmpl
+#include /services/include/configuration-regions.tmpl
+
+
+## {{ model.techShortName }} Node Settings
+
+Adjust the following settings to customize the amount of resources allocated to each node. DC/OS {{ model.techName }}'s [system requirements](http://{{ model.packageName }}.apache.org/doc/latest/operating/hardware.html) must be taken into consideration when adjusting these values. Reducing these values below those requirements may result in adverse performance and/or failures while using the service.
+
+Each of the following settings can be customized under the **node** configuration section.
+
+### Node Count
+
+Customize the `Node Count` setting (default 3) under the **node** configuration section. Consult the {{ model.techName }} documentation for minimum node count requirements.
+
+*   **In DC/OS CLI options.json**: `count`: integer (default: `3`)
+*   **DC/OS web interface**: `NODES`: `integer`
+
+### CPU
+
+You can customize the amount of CPU allocated to each node. A value of 1.0 is equivalent to one full dedicated CPU core on a machine, although all cores are made available via time slicing. Change this value by editing the **cpus** value under the **node** configuration section. Setting this too low will result in throttled tasks.
+
+Please note that each {{ model.techShortName }} node will use an additonal 1.0 CPU for sidecar services such as backup and nodetool. When provisioning 3 CPUS for each {{ model.techShortName }} node, the actual usage will be 4 CPUS, and this should be taken into account when configuring {{ model.techShortName }} to maximize resource utilization on an agent.
+
+*   **In DC/OS CLI options.json**: `cpus`: number (default: `0.5`)
+*   **DC/OS web interface**: `CASSANDRA_CPUS`: `number`
+
+### Memory
+
+You can customize the amount of RAM allocated to each node. Change this value by editing the **mem** value (in MB) under the **node** configuration section. Setting this too low will result in out of memory errors. The `heap.size` setting must also be less than this value to prevent out of memory errors, which can result when the Java Virtual Machine attempts to allocate more memory than is available to the {{ model.techShortName }} process.
```

Comment:
```suggestion
You can customize the amount of RAM allocated to each node. You must change this value by editing the **mem** value (in MB) under the **node** configuration section. Setting this too low will result in out of memory errors. The `heap.size` setting must also be less than this value to prevent out of memory errors, which can result when the Java Virtual Machine attempts to allocate more memory than is available to the {{ model.techShortName }} process.
```

Reply:
no need to mention must here, i am mentioning "you can" to be more clear.